### PR TITLE
only show connected users with a user_id

### PR DESCRIPTION
**Pull request by briangough**
### Description

Minimal fix for https://github.com/overleaf/issues/issues/2179,  only return users with a user_id.

#### Screenshots

NA

#### Related Issues / PRs

https://github.com/overleaf/issues/issues/2179

Longer cleanup PR at https://github.com/overleaf/real-time/pull/76, I won't deploy that today.

### Review

Smallest change to fix this.

#### Potential Impact

Low

#### Manual Testing Performed

- [X] Unit and acceptance tests 

#### Accessibility

NA

### Deployment

NA

#### Deployment Checklist

NA

#### Metrics and Monitoring

NA

### CODE REVIEW

File: test/unit/coffee/ConnectedUsersManagerTests.coffee [hunk 1]
```diff
@@ -124,7 +124,7 @@ describe "ConnectedUsersManager", ->
 
 		it "should return a connected user if there is a user object", (done)->
 			cursorData = JSON.stringify(cursorData:{row:1})
-			@rClient.hgetall.callsArgWith(1, null, {connected_at:new Date(), cursorData})
+			@rClient.hgetall.callsArgWith(1, null, {connected_at:new Date(), user_id: @user._id, last_updated_at: "#{Date.now()}", cursorData})
```
Comment (mserranom) [inline test/unit/coffee/ConnectedUsersManagerTests.coffee]: Perhaps add a new test case, `"should return a not connected user if there is a user object but has no 'user_id'"` ?

File: test/unit/coffee/WebsocketLoadBalancerTests.coffee [hunk 2]
```diff
@@ -17,6 +17,7 @@ describe "WebsocketLoadBalancer", ->
 			"./HealthCheckManager": {check: sinon.stub()}
 			"./RoomManager" : @RoomManager = {eventSource: sinon.stub().returns @RoomEvents}
 			"./ChannelManager": @ChannelManager = {publish: sinon.stub()}
+			"./ConnectedUsersManager": @ConnectedUsersManager = {refreshClient: sinon.stub()}
```
Comment (mserranom) [inline test/unit/coffee/WebsocketLoadBalancerTests.coffee]: What's this change about?

Comment (briangough) [reply test/unit/coffee/WebsocketLoadBalancerTests.coffee]: The tests worked if you had redis running (which apparently we do while running the unit tests on Jenkins) so this needed to be stubbed out.

Comment (mserranom) [reply test/unit/coffee/WebsocketLoadBalancerTests.coffee]: ah nice, thanks

File: test/unit/coffee/ConnectedUsersManagerTests.coffee (hunk 1 above)
Comment (briangough) [reply test/unit/coffee/ConnectedUsersManagerTests.coffee]: Yes, I have a separate PR to clean this up more but I won't deploy that today.

Comment (mserranom) [reply test/unit/coffee/ConnectedUsersManagerTests.coffee]: great 👍

**Pull request merged**